### DISCUSSION TITLE
学习了几天，发现这是一个单页应用程序

### DISCUSSION THREAD

**Discussion by Friedrich-hue** [ChatGPTNextWeb/NextChat#2333] [Q&A]
学习了几天，发现这是一个单页应用程序，如果基于spa框架继续扩展开发的话，会不会出现首屏渲染过慢，，，通常在什么时候会遇到渲染瓶颈，怎么解决，，希望高人指点

**Comment by savokiss** [comment]
这是 next.js 项目，SSR 的
